contributions/catalog/controllers/contribute.py

```diff
@@ -552,9 +552,14 @@ def post_contribution(json_data):
         if result.status_code != 200:
             err_json = parse_response_error(result)
             logging.error("Contribution POST " + json.dumps(err_json))
-            err_msg = str(err_json['status']), err_json['title'], err_json['detail']
-            return False, str("post method fails with error: ") + str(result.status_code) \
-                   + ": " + str(err_msg)
+            if 'reason' in err_json:
```

Comment:
We might have to update the message displayed to the user to make it more user-friendly. Please add a TODO entry for this. Thanks.

Reply:
added TODO

Reply:
Thanks.